c_binding/libata.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
```

Comment:
Just to clarify and to muddy the water, we can use different licenses if the code is executing is a different process.  Thus an out of process plugin can use a different license as it's executing in a separate process.  However, code that executes in the context of the client library should be consistently the same license.